### ISSUE TITLE
Sprint Process?

### ISSUE THREAD

**Issue by mvanveen** [ipfs/py-ipfs#15]
I've been steadily diving head first into IPFS docs and am still getting up to speed.  Along the way, I noticed that there is a [pre-existing weekly sprint process](https://github.com/ipfs/pm/) defined for a lot of the specific current initiatives (go, node, testing + ci, protocol + specs, etc.).

Would this be something that this team would be interested in adopting?

I think one of the benefits is it could solidify strong initial momentum and team to get things going, but it might not fit into everyone's schedule and might not be everyone's favored process.  Since this is an ipfs project, I also think it might be good to stick to their existing defined processes & best practices.

**Comment by bmcorser** [comment]
Yes, not sure about sprints exactly because people have their own schedules. We should adopt a set of rules about code quality, review process and test coverage however:
- PEP8 threshold / styleguide
- No merge without review
- No merge without test coverage

I think it would be great to get a [Homu](http://homu.io/) instance running to decrease PR review management, etc; those Rust guys [know what they're doing](http://huonw.github.io/blog/2015/03/rust-infrastructure-can-be-your-infrastructure/) ...

**Comment by jbenet** [comment]
i believe @amstocker wanted to schedule a py-ipfs hangout. see https://github.com/ipfs/pm/issues/34

**Comment by jbenet** [comment]
the hangouts are useful to:
- review priorities
- decide on tricky issues (not adding new details, just force a decision)
- build momentum
- generate lists of todos that people can pick up

**Comment by jbenet** [comment]
the sprint todos themselves can help people:
- build momentum on what they _want_ to get done, 
- have a realtime venue to show others what they've been up to, get quick feedback, and connect with others
- have a way to see what others have been up to

it's a very low key "i made this stuff!" atmosphere, not a pressured one. do it only if you find it useful

**Comment by amstocker** [comment]
@mvanveen @bmcorser  @jbenet I'm definitely interested in starting a py-ipfs hangout/sprint (also see https://github.com/ipfs/ipfs/issues/112).  It seems like a few people here are really interested in building a python impl of the IPFS protocol so we should have a hangout soon to kick off the project.  I would happy to lead the discussion and I definitely want to contribute to the project, but I am by no means an expert at p2p programming so maybe someone else (@BrendanBenshoof ?) wants to lead?

**Comment by BrendanBenshoof** [comment]
I would be willing to lead that. Timing/Logistics will be fun. Can I start trying to plan a hangout for this weekend?

I think we can do sprints, but I expect they would be 24-48 hour affairs so more people get a chance to participate.

**Comment by BrendanBenshoof** [comment]
#19 
please post availibility

@whyrusleeping @jbenet I'd love if one of you could attend so we can interrogate you about the current state of go-IPFS.

**Comment by whyrusleeping** [comment]
i can probably make a meeting, my schedule is fairly open, but MWF before 1700PST works best for me.

**Comment by NoraCodes** [comment]
I'm in the middle of the college applications process, so this weekend is going to be really busy. I probably won't be available. Sorry.
